test/e2e.go

```diff
@@ -30,7 +30,14 @@ var _ = Describe("E2E Test", func() {
 		err := waitJobReady(context, queueJob.Name)
 		Expect(err).NotTo(HaveOccurred())
 	})
-
+	It("Schedule XQueueJob", func() {
+		context := initTestContext()
+		defer cleanupTestContext(context)
+		rep := clusterSize(context, oneCPU)
+		xqueueJob := createXQueueJob(context, "xqj-1", 2, rep, "busybox", oneCPU)
```

Comment:
Where defined this helper function?

Reply:
Right now, we did not have e2e in CI; it need to make sure e2e passed in your local environment.